source/content/create-custom-upstream.md

```diff
@@ -49,20 +51,20 @@ Choose your preferred Git host:
 
 1. Click **Create Repository**.
 
-1. Copy the repository URL (HTTPS), found in the Quick setup section:
+1. Copy the repository URI (SSH), found in the **Quick setup** section:
```

Comment:
I've called it a URI here, but I'm not sure that it's technically correct

Reply:
I googled this and the [first result was by Daniel Miessler](https://danielmiessler.com/study/difference-between-uri-url/) who is someone I already greatly respect.  

The TL;DR: 
- You're correct to use URI
- "All URLs are URIs, but not all URIs are URLs." 
- If it implies or contains a protocol component (http:// or equiv) then it's a URL.  Looks like github's ssh reference does _not_ contain an explicit protocol indication